setup.cfg

```diff
@@ -26,9 +26,9 @@ classifiers =
 include_package_data = True
 install_requires =
     arrow # pyup: ignore
-    Django<3.0.0 # pyup: ignore
+    Django # pyup: ignore
```

Comment:
We always install the latest django version? Should we not use a fixed version?

Reply:
In fact we don't pin our dependencies unless necessary. It was necessary until now to limit Django to v2 but this constraint is now gone :tada: